components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -582,6 +580,28 @@ private String getCaptchaParams(String tenantDomain) {
                     log.debug("Enforcing recaptcha always for the basic authentication is not enabled.");
                 }
             }
+
+            // Resend confirmation captcha config
+            connectorConfigs = BasicAuthenticatorDataHolder.getInstance().getIdentityGovernanceService()
```

Comment:
Use the service call at line 564 to get this config as well in the same request.